src/test/resources/unsolvedmethodtypecorrect/expected/com/example/Simple.java

```diff
@@ -1,15 +1,13 @@
 package com.example;
 
+import org.example.LocalVariables;
 import org.example.Foo;
 
 class Simple {
     void bar() {
         Foo f = new Foo();
         f.getLocals().set(0);
         baz();
-    }
-
-    void baz() {
-        throw new java.lang.Error();
+        final LocalVariables locals = f.getLocals();
```

Comment:
This change doesn't look correct to me. The original input file is:
```
package com.example;

import org.example.LocalVariables;
import org.example.Foo;

class Simple {
    // Target method.
    void bar() {
        Foo f = new Foo();
        f.getLocals().set(0);
        baz();
    }

    void baz() {
        // To trigger JavaTypeCorrect.
        Foo f = new Foo();
        final LocalVariables locals = f.getLocals();
    }
}
```

Putting the contents of the `baz()` method into the `bar()` method definitely seems incorrect.

Reply:
> I modified the test case (the input as well) to move the line into bar() so that we'd still be able to test for the same thing, which is why the output here looks the way it does.

Thanks, I missed that you'd changed both.